assets/blocks/blocks-setup.js

```diff
@@ -0,0 +1,16 @@
+import { registerBlockType, updateCategory } from '@wordpress/blocks';
+
+import { SenseiIcon } from '../icons';
+
+const blocksSetup = ( blocks ) => {
```

Comment:
How about something like `registerSenseiBlocks`?

Reply:
I used a more generic name because we were also setting the category icon, and we can to other stuff there, but I think this name works well too.

Updated in https://github.com/Automattic/sensei/pull/3885/commits/19ed087d9f7f780aaebefa0c3e2126211cbb362a